app/stores/interactive-filters.tsx

```diff
@@ -239,24 +232,13 @@ export const InteractiveFiltersProvider = ({
     );
   }, [chartConfigs]);
 
-  const timeRange = state.dashboardFilters?.timeRange;
-  const dataFilters = state.dashboardFilters?.dataFilters;
-
   const ctxValue = useMemo(
     () => ({
       potentialTimeRangeFilterIris,
-      timeRange,
       potentialDataFilterIris,
-      dataFilters,
       stores,
     }),
-    [
-      potentialTimeRangeFilterIris,
-      timeRange,
-      potentialDataFilterIris,
-      dataFilters,
-      stores,
-    ]
+    [potentialTimeRangeFilterIris, potentialDataFilterIris, stores]
```

Comment:
question: Why this change ? Was it causing a problem ? I feel like from the consumer side, it makes it more complex and less flexible. I think I had "hidden" it behind the context so that we could be flexible in the context and every consumer do not have to worry where or how the filters are stored/computed.

Reply:
Discussed offline: it was to keep the separation between "static",  config filters and "dynamic", interactive filters more clear. We could still improve this by introducing e.g. `useConfigDashboardInteractiveFilters` and `useConfigInteractiveFilters` to hide the details of accessing them.